range_sensor_layer/src/range_sensor_layer.cpp

```diff
@@ -382,51 +391,58 @@ void RangeSensorLayer::updateBounds(double robot_x, double robot_y, double robot
   min_x_ = min_y_ = std::numeric_limits<double>::max();
   max_x_ = max_y_ = std::numeric_limits<double>::min();
 
-  if (!enabled_)
-  {
+  if (!enabled_) {
     current_ = true;
     return;
   }
-  
-  if (buffered_readings_ == 0)
-  {
+
+  if (buffered_readings_ == 0) {
     if (no_readings_timeout_ > 0.0 &&
-        (ros::Time::now() - last_reading_time_).toSec() > no_readings_timeout_)
-    {
-      ROS_WARN_THROTTLE(2.0, "No range readings received for %.2f seconds, " \
-                             "while expected at least every %.2f seconds.",
-               (ros::Time::now() - last_reading_time_).toSec(), no_readings_timeout_);
+        (ros::Time::now() - last_reading_time_).toSec() >
+            no_readings_timeout_) {
+      ROS_WARN_THROTTLE(2.0,
+                        "No range readings received for %.2f seconds, "
+                        "while expected at least every %.2f seconds.",
+                        (ros::Time::now() - last_reading_time_).toSec(),
+                        no_readings_timeout_);
       current_ = false;
     }
   }
-
 }
 
-void RangeSensorLayer::updateCosts(costmap_2d::Costmap2D& master_grid, int min_i, int min_j, int max_i, int max_j)
-{
+void RangeSensorLayer::updateCosts(costmap_2d::Costmap2D &master_grid,
+                                   int min_i, int min_j, int max_i, int max_j) {
   if (!enabled_)
     return;
 
-  unsigned char* master_array = master_grid.getCharMap();
+  unsigned char *master_array = master_grid.getCharMap();
   unsigned int span = master_grid.getSizeInCellsX();
-  unsigned char clear = to_cost(clear_threshold_), mark = to_cost(mark_threshold_);
+  unsigned char clear = to_cost(clear_threshold_),
+                mark = to_cost(mark_threshold_);
 
-  for (int j = min_j; j < max_j; j++)
-  {
+  for (int j = min_j; j < max_j; j++) {
     unsigned int it = j * span + min_i;
-    for (int i = min_i; i < max_i; i++)
-    {
+    for (int i = min_i; i < max_i; i++) {
       unsigned char prob = costmap_[it];
       unsigned char current;
-      if(prob==costmap_2d::NO_INFORMATION){
+
+      if (prob == costmap_2d::NO_INFORMATION) {
         it++;
         continue;
-      }
-      else if(prob>mark)
+      } else if (prob > mark) {
         current = costmap_2d::LETHAL_OBSTACLE;
-      else if(prob<clear)
+        costmap_[it]--;
```

Comment:
use prefix operators unless you need to postfix semantics.

Reply:
done